acos_client/v30/slb/service_group.py

```diff
@@ -50,19 +50,26 @@ def get(self, name, **kwargs):
         return self._get(self.url_prefix + name, **kwargs)
 
     def _set(self, name, protocol=None, lb_method=None, hm_name=None,
-             update=False, **kwargs):
+             hm_disable=0, update=False, **kwargs):
```

Comment:
_set is an internal method and should only be called by the public method set.  Again, not sure why I didn't just leverage the existing kwargs.